src/main/io/beeper.h

```diff
@@ -102,7 +102,7 @@ STATIC_ASSERT(BEEPER_ALL < sizeof(uint32_t) * 8, "BEEPER bits exhausted");
     | BEEPER_GET_FLAG(BEEPER_ARMING_GPS_NO_FIX) \
     )
 #ifdef USE_RACE_PRO
-#define DEFAULT_DSHOT_BEACON_OFF_FLAGS BEEPER_GET_FLAG(BEEPER_SILENCE)
+#define DEFAULT_DSHOT_BEACON_OFF_FLAGS BEEPER_SILENCE
```

Comment:
This code path was never tested, suggest reverting
```suggestion
#define DEFAULT_DSHOT_BEACON_OFF_FLAGS BEEPER_GET_FLAG(BEEPER_RX_LOST)
```